src/main/java/seedu/address/model/person/Runner.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.UniqueTagList;
+
+/**
+ * Represents a Person in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Runner extends Person {
+
+    private final Name name;
+    private final Phone phone;
+    private final Email email;
+    private final Address address;
+
+    private final UniqueTagList tags;
+
+    private List<Customer> assignedCustomers;
+
+
+    /**
+     * Every field must be present and not null.
+     */
+    public Runner(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
+        requireAllNonNull(name, phone, email, address, tags);
+        this.name = name;
+        this.phone = phone;
+        this.email = email;
+        this.address = address;
+        this.tags = new UniqueTagList(tags);
+
+    }
```

Comment:
Could we can call the `Person` constructor to do the assignment of attributes inherited from the `Person` parent class?